File: app/src/main/java/name/monwf/customiuizer/utils/ModData.java
package name.monwf.customiuizer.utils;

public class ModData {

	public enum ModCat {
		pref_key_system,
		pref_key_launcher,
		pref_key_controls,
		pref_key_various
	}

	public String title;
	public String breadcrumbs;
	public String key;
	public ModCat cat;
	public String sub;
	public int order;
}

File: app/src/main/java/tv/withaibuild/customiuizer/utils/ModData.kt
package tv.withaibuild.customiuizer.utils

class ModData {

    enum class ModCat {
        pref_key_system,
        pref_key_launcher,
        pref_key_controls,
        pref_key_various
    }

    @JvmField
    var title: String = ""

    @JvmField
    var breadcrumbs: String = ""

    @JvmField
    var key: String = ""

    @JvmField
    var cat: ModCat = ModCat.pref_key_system

    @JvmField
    var sub: String = ""

    @JvmField
    var order: Int = 0
}
